traveler-ios-ui/TravelerKitUI/Views/FlightDetailCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FlightDetailCell.swift
+//  TravelerKitUI
+//
+//  Created by Rakin Hoque on 2019-12-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+
+class FlightDetailCell: UITableViewCell {
```

Comment:
Bad naming, view can't be model aware. The cell shuold have a generic name with generic labels. The same is true for the nibs.